include/receiver_cadmium.hpp

```diff
@@ -1,4 +1,13 @@
-/**
+/** \brief This header file implements the Receiver class.
+*
+* The receiver receives the message on input port
+* and sends acknowledge after fixed delay on output port.
+*
+* The receiver has two phases: passive and active.
+* It is in passive phase initially. When it receives
+* a packet, it will go to active phase, send out
+* the acknowledgement and go to passive state again.
+*
```

Comment:
I would suggest to separate this comment section into two. 
First section can contain the brief and detailed description. 
Second section can include developer details.

Reply:
solved